### PR TITLE
Remove transcript preferences

### PR DESCRIPTION
**Pull request by mushtaqak**
This PR adds an api end point which will let us remove course-wide transcript preferences. This would mainly come in handy when user selects a **N/A** option from Video Uploads page.

### CODE REVIEW

File: edxval/tests/test_api.py [hunk 1]
```diff
@@ -1731,6 +1731,21 @@ def test_get_transcript_preferences(self):
...
+        # Verify that transcript preferences exist
+        transcript_preferences = api.get_transcript_preferences(self.course_id)
+        self.assertIsNotNone(transcript_preferences)
+
+        # Remove course wide transcript preferances.
+        api.remove_transcript_preferences(self.course_id)
+
+        # Verify now transcript preferances no longer exist.
+        transcript_preferences = api.get_transcript_preferences(self.course_id)
+        self.assertIsNone(transcript_preferences)
```
Comment (muzaffaryousaf) [inline edxval/tests/test_api.py]: can we use different course id to raise does't exist for coverage.

**Comment by muzaffaryousaf** [comment]
Don't forget to squash & address my single comment.

File: edxval/api.py [hunk 2]
```diff
@@ -266,6 +266,22 @@ def create_or_update_transcript_preferences(course_id, **preferences):
...
+    Arguments:
+        course_id(str): course id
+    """
+    transcript_preference = None
+    try:
+        transcript_preference = TranscriptPreference.objects.get(course_id=course_id)
+    except ObjectDoesNotExist:
+        return
+
+    transcript_preference.delete()
```
Comment (Qubad786) [inline edxval/api.py]: move this into `try` block, and remove unnecessary `transcript_preference=None` on #L276

File: edxval/api.py [hunk 3]
```diff
@@ -266,6 +266,22 @@ def create_or_update_transcript_preferences(course_id, **preferences):
...
+    """
+    Deletes course-wide transcript preferences.
+
+    Arguments:
+        course_id(str): course id
+    """
+    transcript_preference = None
+    try:
+        transcript_preference = TranscriptPreference.objects.get(course_id=course_id)
+    except ObjectDoesNotExist:
```
Comment (Qubad786) [inline edxval/api.py]: `TranscriptPreference.DoesNotExist` for readability.

File: edxval/api.py [hunk 4]
```diff
@@ -266,6 +266,22 @@ def create_or_update_transcript_preferences(course_id, **preferences):
...
+        return
```
Comment (Qubad786) [inline edxval/api.py]: we can just pass this?

**Comment by mushtaqak** [comment]
@muzaffaryousaf @Qubad786 Addressed feedbacks :)

**Pull request merged**